Code/src/main/java/nl/utwente/group10/ui/exceptions/InvalidAnchorException.java

```diff
@@ -0,0 +1,5 @@
+package nl.utwente.group10.ui.exceptions;
+
+public class InvalidAnchorException extends RuntimeException {
```

Comment:
In ConnectionCreationManager I believe, although that should never happen, but does happen very rarely, don't know why :(